### PR TITLE
meta: Add core-ui as owners of all components

### PR DESCRIPTION
**Pull request by evanpurkhiser**
We'll see if this ends up being too much. Maybe needs to have a bit of discussion on this PR?

Do we block peoples PRs because of this?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -69,6 +69,8 @@ build-utils/        @getsentry/owners-js-build
 /static/images/                                                 @getsentry/design
 /docs-ui/components/                                            @getsentry/design
 
+# Frontend
+/static/app/components  @getsentry/core-ui
```
Comment (markstory) [inline .github/CODEOWNERS:73]: Should you take /docs-ui/components/ too?

Comment (EvanPurkhiser) [reply .github/CODEOWNERS:73]: good call

**Comment by billyvg** [comment]
It won't block PRs, just tags you as reviewer

**Comment by EvanPurkhiser** [comment]
I like that

**Review by markstory — APPROVED** [review]

**Pull request merged**